packages/gasket-cli/lib/bin/boot.js

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env node
-
 const { join } = require('path');
-const location = ['@gasket', 'cli', 'bin', 'run'];
+const location = ['@gasket', 'cli', 'lib', 'bin', 'run'];
```

Comment:
Sorry to churn on this... but it would probably be best for this lookup path to remain the same for backward (and forward) compatibility. For instance, if v7 cli is globally installed, it should still try to use v6 of the local app, and visa-versa.

Perhaps the package structure should look more like this:

```
./bin/run.js
./bin/boot.js
./lib/init.js
```